assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -159,6 +167,9 @@ function MediaPane(props) {
         resource,
         mediaPickerEl.sizes?.medium?.url || mediaPickerEl.url
       );
+
+      // Update the video poster on the ride side bar (i.e. Accessibility)
```

Comment:
```suggestion
      // Upload video poster and update media element afterwards,
      // so that the poster will correctly show up in places like the Accessibility panel.
```

Reply:
Updated.